### PR TITLE
add param bench to CI test

### PR DESCRIPTION
**Pull request by kingchc**
Summary: add param bench to CI test

Differential Revision: D30207221

### CODE REVIEW

File: .github/workflows/main.yaml [hunk 1]
```diff
@@ -21,7 +22,7 @@ jobs:
     - name: Get UCX
       run: |
         git clone ${OPENUCX_LINK} /tmp/ucx
-        cd /tmp/ucx &&
+        cd /tmp/ucx && git checkout fd3e116e7f9ff61133918e1b06076d5d31f1ba6c
```
Comment (kingchc) [inline .github/workflows/main.yaml:25]: Looks like switch to older version UCX make all tests passed (except Alltoall due to recent API change, see #21 ) (cc @bureddy)

**Comment by lappazos** [comment]
https://github.com/openucx/ucc/pull/287 might solve the problem with UCX version, please wait for this PR to merge and try again, with AlltoAll as well

**Comment by lappazos** [comment]
@kingchc why not close this PR and stick with PR 29?

**Comment by kingchc** [comment]
> @kingchc why not close this PR and stick with PR 29?

sounds good, let me close this one and go with #29

**Pull request closed without merging**